rpm/gtest.spec

```
@@ -76,12 +79,14 @@ Requires:       libgmock = %{version}-%{release}
 Documentation files for libgmock.
 
 %prep
-%setup -q -n %{name}-%{version}/%{name}
+%autosetup -q -n %{name}-%{version}/%{name}
```

In such case you need to pass additional argument for patch:
```suggestion
%autosetup -p1 -n %{name}-%{version}/%{name}
```
And you don't need any more to specify all the patches explicitly: https://rpm.org/user_doc/autosetup.html